src/vs/platform/windows/electron-main/windowImpl.ts

```diff
@@ -287,11 +287,26 @@ export abstract class BaseWindow extends Disposable implements IBaseWindow {
 		return !!this.documentEdited;
 	}
 
-	focus(options?: { force: boolean }): void {
-		if (isMacintosh && options?.force) {
+	focus(options?: { force: boolean; notify: boolean }): void {
+		const hasAnyFocusedWindow = electron.BrowserWindow.getAllWindows().some(w => w.isFocused());
+
+		if (hasAnyFocusedWindow) {
+			this._focusWindow();
+		} else if (options?.notify) {
+			if (isMacintosh) {
```

Comment:
What about Linux?

Reply:
Linux doesn't have any mechanism for this from what I can tell